web/src/main/java/edu/softserve/zoo/controller/rest/impl/EmployeeRestController.java

```diff
@@ -31,4 +34,10 @@ public EmployeeRestController() {
     protected Service<Employee> getService() {
         return employeeService;
     }
+
+    @Override
+    @RequestMapping(method = RequestMethod.GET)
+    public List<EmployeeDto> getAll() {
+        return super.getAll();
```

Comment:
You do not need to return all employees. Assuming you have several admins, several users for each other role. Why do you need to get all?